### PR TITLE
CV-1960 Adds Unbounce custom global namespace

### PR DESCRIPTION
**Pull request by kalupa**
Adds a custom unbounce namespace to minimize collisions with customer scripts containing snowplow as well.

### CODE REVIEW

File: src/js/init.js [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
```
Comment (kalupa) [inline src/js/init.js:31]: Prettier was not being run on this file before, it seems.

File: src/js/init.js [hunk 2]
```diff
@@ -37,17 +37,28 @@
...
  * Get the name of the global input function
  */
-
 var snowplow = require('./snowplow'),
-	queueName,
-	queue,
-	windowAlias = window;
+  queueName,
+  queue,
+  windowAlias = window
```
Comment (kalupa) [inline src/js/init.js:43]: prettier reformat

File: src/js/init.js [hunk 3]
```diff
@@ -37,17 +37,28 @@
...
+  windowAlias.GlobalSnowplowNamespace.length > 0
+) {
+  queueName = windowAlias.GlobalSnowplowNamespace.shift()
+  queue = windowAlias[queueName]
+  queue.q = new snowplow.Snowplow(queue.q, queueName)
 } else {
-	windowAlias._snaq = windowAlias._snaq || [];
-	windowAlias._snaq = new snowplow.Snowplow(windowAlias._snaq, '_snaq');
+  windowAlias._snaq = windowAlias._snaq || []
+  windowAlias._snaq = new snowplow.Snowplow(windowAlias._snaq, '_snaq')
```
Comment (kalupa) [inline src/js/init.js:63]: prettier reformat as well

File: src/js/init.js [hunk 4]
```diff
@@ -37,17 +37,28 @@
...
-if (windowAlias.GlobalSnowplowNamespace && windowAlias.GlobalSnowplowNamespace.length > 0) {
-	queueName = windowAlias.GlobalSnowplowNamespace.shift();
-	queue = windowAlias[queueName];
-	queue.q = new snowplow.Snowplow(queue.q, queueName);
+if (
+  windowAlias.UnbounceSnowplowNamespace &&
+  windowAlias.UnbounceSnowplowNamespace.length > 0
+) {
+  // Unbounce Namespace
+  queueName = windowAlias.GlobalSnowplowNamespace.shift()
```
Comment (stevefisherdev) [inline src/js/init.js]: I think this line should be updated to `UnbounceSnowplowNamespace`?

**Review by stevefisherdev — APPROVED** [review]
LGTM

File: .travis.yml [hunk 5]
```diff
@@ -23,44 +23,45 @@ install:
...
+#   - secure: JwP5MuqZ6k8FlHg8AnzYTrCZByJLnlu0to2srEwfVOhW1efGTEmtWf/SLlplYHcdelsFakdiSbD3BkZyEqpe/2/AIb56KLAqtE2Ng3WS8Vz/80yknxII0xcSPzAZdhZYRUJlbIrA1Ua3XGgirBenjn7ILFXNX/qLgvwJ1uLKCPo=
+#   # AWS_ACCESS_KEY
+#   - secure: AEvIKW1H/DPU5yPKfRPHY9XpBCjXGtwVqpQERx0wYGwV8j9mzz8EMMrx6yPMaLOkeBMcaH/db3aXEePHWAlnxrTuUuhwwr9A7sihaaAwZ9GllUlUEJzfXeBcCM9rhsooY2roMzJM3Ener1W+yKHThagYxUZ4PyZrPJCMMzbc0qw=
+#   # AWS_SECRET_KEY
+#   - secure: C+PTon3PsELG2bFlAZ5qfinWVih8XxlZV45Gk2W5i2CK1AROJ60lnfhgYZM4YidIRcA/V/zbm8tDgZbA8i/MVKcO5f1qbD5Skyo2fEFRlXoSWhtwaScViihN338U/SsEviKThIGSSLgdfsQ0CW7TYxvetFSwXFyZPPrYrenF3dk=
+#   # NPM_AUTH_TOKEN
+#   - secure: PwIU7BHXhpHUhFh/sxRO1o9awiTKJ5PNrSgeTytp2CcE22q0yx/q5cKJp/gPWiY0l0sKSrNjcV/s0hifg3d8q4YcTa3JEK6Ap71ehOsCIJQD897cgTMqgawkdspeyZL8OPPAs0hMwHFAASm7RDTe4zz1rYMFs2+DYLbIKu0Ki8I=
+#   - AWS_S3_BUCKET=snowplow-static-js
+#   - AWS_REGION=eu-west-1
+#   - AWS_CF_DISTRIBUTION=E1RICD7QD0F2KE
```
Comment (kalupa) [inline .travis.yml:67]: I've left this in the file, commented, to minimize the irritation of keeping our fork up to date

**Pull request merged**